libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -726,7 +716,8 @@ size_t CodeTransform::variableHeightDiff(Scope::Variable const& _var, YulString
 			to_string(heightDiff - limit) +
 			" slot(s) too deep inside the stack."
 		);
-		BOOST_THROW_EXCEPTION(m_stackErrors.back());
+		// TODO: maybe make this return something special that results in producing INVALID instead.
```

Comment:
We could also set an "invalid" flag on `m_assembly` which causes the assembly to throw when the bytecode is requested.